extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -67,4 +67,7 @@ message PluginConfig {
   // metrics for HTTP traffic. Defaults to false (request and response size
   // metrics are enabled).
   bool disable_http_size_metrics = 8;
+
+  // Optional. Allows enabling log compression for stackdriver access logs.
+  bool enable_log_compression = 9;
```

Comment:
Can you make it a [BoolValue](https://github.com/protocolbuffers/protobuf/blob/d4ec70fdd3380e7113e9e3c23cef2bea50dbf2e7/src/google/protobuf/wrappers.proto#L104), so that we can flip the default behavior of unset easily?